### DISCUSSION TITLE
[feature request] Add control for allowed granularities in Time grain filter

### DISCUSSION THREAD

**Discussion by kamalkeshavani-aiinside** [apache/superset#21525] [Ideas]
Currently in 'Time Grain' type native filter, it will show all available granularities to the end user in the dropdown.

We would like to control/limit the visible granularities in a dashboard filter for end user.
<img width="252" alt="image" src="https://user-images.githubusercontent.com/74634977/191163680-dbe84da3-7550-4a3a-9927-0750bf139c79.png">

**Comment by sfirke** [comment]
I'm looking through these issues and agree with you.  Is this the same as https://github.com/apache/superset/discussions/20191 ?

**Reply by kamalkeshavani-aiinside** [reply]
Yes, looks same to me.
